libminifi/test/unit/ProcessSessionTests.cpp

```diff
@@ -126,3 +132,44 @@ TEST_CASE("ProcessSession::read can read zero length flowfiles without crash", "
   ContentRepositoryDependentTests::testReadFromZeroLengthFlowFile(std::make_shared<core::repository::VolatileContentRepository>());
   ContentRepositoryDependentTests::testReadFromZeroLengthFlowFile(std::make_shared<core::repository::FileSystemRepository>());
 }
+
+struct VolatileFlowFileRepositoryTestAccessor {
+  METHOD_ACCESSOR(flush);
+};
+
+class TestVolatileFlowFileRepository : public core::repository::VolatileFlowFileRepository {
+ public:
+  explicit TestVolatileFlowFileRepository(std::string name) : core::SerializableComponent(std::move(name)) {}
```

Comment:
why is this not
```suggestion
  explicit TestVolatileFlowFileRepository(std::string name) : core::repository::VolatileFlowFileRepository(std::move(name)) {}
```
?

Reply:
`SerializableComponent` is a virtual base class without default initializer, so we have to directly initialize it in every derived class